### PR TITLE
update outdated comments

### PR DESCRIPTION
**Pull request by knightXun**
update outdated comments

### CODE REVIEW

File: source/server/listener_manager_impl.cc [hunk 1]
```diff
@@ -49,11 +49,10 @@ std::vector<Network::FilterFactoryCb> ProdListenerComponentFactory::createNetwor
     const Protobuf::RepeatedPtrField<envoy::api::v2::listener::Filter>& filters,
     Configuration::FactoryContext& context) {
   std::vector<Network::FilterFactoryCb> ret;
-  for (ssize_t i = 0; i < filters.size(); i++) {
+  for (auto i = 0; i < filters.size(); i++) {
```
Comment (junr03) [inline source/server/listener_manager_impl.cc]: qq why the change to auto here?

Comment (knightXun) [reply source/server/listener_manager_impl.cc]: i want make it  conform to c++ 11

**Comment by mattklein123** [comment]
@knightXun thanks for your contribution, but I'm going to go ahead and close this. Some of these changes are changes for change sake, and some of the comment change are incorrect (the original comment is correct). PRs are welcome, but to avoid review overhead please open PRs for things that are definitely broken and not just a matter of taste. Thank you!

**Comment by knightXun** [comment]
thanks
